Blocks/HTTP/request.h

```diff
@@ -51,7 +51,7 @@ constexpr static auto HasRespondViaHTTP(int)
 }
 
 // The only parameter to be passed to HTTP handlers.
-struct Request final {
+struct Request final : CurrentSuper {
```

Comment:
To be able to `std::move()` a `Request` itself, without a wrapper class, contained in the `std::unique_ptr<CurrentSuper, CurrentSuperDeleter>` which is within the MM[P]Q of RipCurrent.

In other words, a RipCurrent artifact of this code, sorry it's part of this PR.